pygmt/datasets/earth_relief.py

```diff
@@ -81,7 +83,10 @@ def _is_valid_resolution(resolution):
     pygmt.exceptions.GMTInvalidInput: Invalid Earth relief resolution '01s'.
 
     """
-    valid_resolutions = ["{:02d}m".format(res) for res in [60, 30, 10, 5, 2, 1]]
+    valid_resolutions = ["01d"]
+    valid_resolutions.extend(
+        ["{:02d}m".format(res) for res in [60, 30, 20, 15, 10, 6, 5, 4, 3, 2, 1]]
+    )
     valid_resolutions.extend(["{:02d}s".format(res) for res in [30, 15]])
```

Comment:
```suggestion
    valid_resolutions.extend(
        [f"{res:02d}m" for res in [60, 30, 20, 15, 10, 6, 5, 4, 3, 2, 1]]
    )
    valid_resolutions.extend([f"{res:02d}s" for res in [30, 15]])
```

Could use an f-string here. Also, we'll need to update this to include the 3 and 1 arc-second grids once GMT 6.1.0 comes out. For reference to future maintainers, valid resolutions are at https://docs.generic-mapping-tools.org/latest/datasets/earth_relief.html